src/Aspire.Hosting/api/Aspire.Hosting.cs

```diff
@@ -1010,9 +1215,23 @@ public ConnectionStringReference(IResourceWithConnectionString resource, bool op
 
         public IResourceWithConnectionString Resource { get { throw null; } }
 
+        System.Threading.Tasks.ValueTask<string?> IValueProvider.GetValueAsync(ValueProviderContext context, System.Threading.CancellationToken cancellationToken) { throw null; }
+
         System.Threading.Tasks.ValueTask<string?> IValueProvider.GetValueAsync(System.Threading.CancellationToken cancellationToken) { throw null; }
     }
 
+    public sealed partial class ContainerCertificatePathsAnnotation : IResourceAnnotation
+    {
+        public static System.Collections.Immutable.ImmutableList<string> DefaultCertificateBundlePaths;
+        public static System.Collections.Immutable.ImmutableList<string> DefaultCertificateDirectoriesPaths;
+        public const string DefaultCustomCertificatesDestination = "/usr/lib/ssl/aspire";
```

Comment:
Will this ever change? Say in a different Unix like environment that doesn't use `/usr`? Maybe it shouldn't be a `const string` but instead a readonly property.